junebug/channel.py

```diff
@@ -201,6 +201,19 @@ def get_all(cls, redis):
         channels = yield redis.smembers('channels')
         returnValue(channels)
 
+    @classmethod
+    @inlineCallbacks
+    def start_all_channels(cls, redis, config, parent):
+        '''Ensures that all of the stored channels are running'''
+        for id in (yield cls.get_all(redis)):
+            try:
+                yield cls.from_id(redis, config, id, parent)
```

Comment:
Would there possibly be cases where we have a channel in-memory but it is not started?